pkg/controller/schedule_controller.go

```diff
@@ -199,11 +199,10 @@ func (c *scheduleReconciler) checkIfBackupInNewOrProgress(schedule *velerov1.Sch
 
 	for _, backup := range backupList.Items {
 		if backup.Status.Phase == velerov1.BackupPhaseNew || backup.Status.Phase == velerov1.BackupPhaseInProgress {
+			log.Debugf("Schedule %s/%s still has backups are in InProgress or New state, skip submitting backup to avoid overlap.", schedule.Namespace, schedule.Name)
```

Comment:
NIT should it be "... still has backups **that** are in InProgress..." or "... still has backups in InProgress..."?

Reply:
Thanks for the quick review.
I am not a english speaker.
Your suggestion is make sense to me.

Reply:
Rebased.